libhb/handbrake/preset_builtin.h

```diff
@@ -98,6 +98,7 @@ const char hb_builtin_presets_json[] =
 "                    \"VideoOptionExtra\": \"\", \n"
 "                    \"VideoPreset\": \"veryfast\", \n"
 "                    \"VideoProfile\": \"main\", \n"
+"                    \"VideoQSVAsyncDepth\": 4, \n"
```

Comment:
QSVAsyncDepth = 6 is used on latest Intel HW. Maybe is better not specify this parameter and it will be selected correctly by default depending on HW generation.

Reply:
Okay, I forget which commit originally set that. Can certainly be changed.

@jstebbins @sr55 Is `VideoQSVAsyncDepth == 0` equivalent to automatic?

Reply:
default/automatic will be used when No such option, 
see https://github.com/HandBrake/HandBrake/blob/6fbe60b266fed099f49616ecef34db88b6bb3f4b/libhb/preset.c#L1792

Reply:
Ah, yes. I was hoping I could use `0` or `-1` rather than omit it entirely. It looks like all presets have it set to `4` currently, seems we should update all of them following https://github.com/HandBrake/HandBrake/commit/73681c07cef1b65278a382a01dc27cdf09d04970, not necessarily related to this PR specifically.